src/Service/Ecr/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## NOT RELEASED
 
+### Added
+
+- AWS api-change: TODO
```

Comment:
```suggestion
- AWS api-change: Add support for the `il-central-1` region
```

Reply:
we usually don't suffix these with `AWS api-change: `